build.sh

```diff
@@ -27,11 +27,11 @@ popd > /dev/null
 # Parse project config
 PROJECTCONF=$ROOT/conf/project.yml
 eval $(parse_yaml $PROJECTCONF)
-ALIASFILE=${project_name}.aliases.drushrc.php
-ALIASTARGET=$HOME/.drush/$ALIASFILE
+ALIASFILE=self.site.yml
```

Comment:
Alias filename should be unique across projects, otherwise only the first one will work as later projects are unable to create their alias file.

Reply:
Ok, will push soon.